go/eks/simple-cluster/constructs/addons/ebs-csi-driver.go

```diff
@@ -0,0 +1,257 @@
+				Effect: awsiam.Effect_ALLOW,
+				Actions: &[]*string{
+					jsii.String("ec2:CreateVolume"),
+				},
+				Resources: &[]*string{
+					jsii.String("*"),
+				},
+				Conditions: &map[string]interface{}{
+					"StringLike": awscdk.NewCfnJson(stack, jsii.String("CfnJson-EBSCSI-S3"), &awscdk.CfnJsonProps{
+						Value: map[string]string{
+							"aws:RequestTag/kubernetes.io/cluster/*": "owned",
+						},
+					}),
+				},
+			}),
+			awsiam.NewPolicyStatement(&awsiam.PolicyStatementProps{
+				Effect: awsiam.Effect_ALLOW,
+				Actions: &[]*string{
+					jsii.String("ec2:DeleteVolume"),
+				},
+				Resources: &[]*string{
+					jsii.String("*"),
+				},
+				Conditions: &map[string]interface{}{
+					"StringLike": awscdk.NewCfnJson(stack, jsii.String("CfnJson-EBSCSI-S4"), &awscdk.CfnJsonProps{
+						Value: map[string]string{
+							"ec2:ResourceTag/ebs.csi.aws.com/cluster": "true",
+						},
+					}),
+				},
+			}),
+			awsiam.NewPolicyStatement(&awsiam.PolicyStatementProps{
+				Effect: awsiam.Effect_ALLOW,
+				Actions: &[]*string{
+					jsii.String("ec2:DeleteVolume"),
+				},
+				Resources: &[]*string{
+					jsii.String("*"),
+				},
+				Conditions: &map[string]interface{}{
+					"StringLike": awscdk.NewCfnJson(stack, jsii.String("CfnJson-EBSCSI-S5"), &awscdk.CfnJsonProps{
+						Value: map[string]string{
+							"ec2:ResourceTag/CSIVolumeName": "*",
+						},
+					}),
+				},
+			}),
+			awsiam.NewPolicyStatement(&awsiam.PolicyStatementProps{
+				Effect: awsiam.Effect_ALLOW,
+				Actions: &[]*string{
+					jsii.String("ec2:DeleteVolume"),
+				},
+				Resources: &[]*string{
+					jsii.String("*"),
+				},
+				Conditions: &map[string]interface{}{
+					"StringLike": awscdk.NewCfnJson(stack, jsii.String("CfnJson-EBSCSI-S6"), &awscdk.CfnJsonProps{
+						Value: map[string]string{
+							"ec2:ResourceTag/kubernetes.io/cluster/*": "owned",
+						},
+					}),
+				},
+			}),
+			awsiam.NewPolicyStatement(&awsiam.PolicyStatementProps{
+				Effect: awsiam.Effect_ALLOW,
+				Actions: &[]*string{
+					jsii.String("ec2:DeleteSnapshot"),
+				},
+				Resources: &[]*string{
+					jsii.String("*"),
+				},
+				Conditions: &map[string]interface{}{
+					"StringLike": awscdk.NewCfnJson(stack, jsii.String("CfnJson-EBSCSI-S7"), &awscdk.CfnJsonProps{
+						Value: map[string]string{
+							"ec2:ResourceTag/CSIVolumeSnapshotName": "*",
+						},
+					}),
+				},
+			}),
+			awsiam.NewPolicyStatement(&awsiam.PolicyStatementProps{
+				Effect: awsiam.Effect_ALLOW,
+				Actions: &[]*string{
+					jsii.String("ec2:DeleteSnapshot"),
+				},
+				Resources: &[]*string{
+					jsii.String("*"),
+				},
+				Conditions: &map[string]interface{}{
+					"StringLike": awscdk.NewCfnJson(stack, jsii.String("CfnJson-EBSCSI-S8"), &awscdk.CfnJsonProps{
+						Value: map[string]string{
+							"ec2:ResourceTag/ebs.csi.aws.com/cluster": "true",
+						},
+					}),
+				},
+			}),
+		},
+	})
+
+	// For arm64/aarch64 cluster
```

Comment:
NIT: not a huge fan of this being just *commented out*.

To be completely honest, I'd love to see these rewritten in Typescript and added into the EKS constructs in the core CDK since they're so useful, *but*, I can't have everything today. @rix0rrr is this a feature the team would be interested in having?